examples/error-handling/error_handling.bal

```diff
@@ -1,36 +1,40 @@
 import ballerina/io;
 
+// Define a record to represent error details.
 type AccountNotFoundErrorData record {
-    string accountID;
+    int accountID;
 };
 
-// Extended error types can be declared from `error` type. Thus `error` can be extended for single level.
+// User defined `error` types can be introduced based on the `error` basic type.
```

Comment:
```suggestion
// User-defined error types can be introduced via the `error` basic type.
```

Reply:
@praneesha - Is there any specific reason to use **'** instead of **`**?

Reply:
It is not very common to use the backtick. Also, we do not have the opening and closing shapes in the backtick. 

Thus, it is simpler to use the apostrophe unless we have a technical best practice in coding to use the backtick.

Reply:
Seems like we need to use the backtick to render keywords from the markdown file. Sorry, it was my bad. Will resolve the suggestions related to this point.

Reply:
IMO, "based on" is more accurate than "via" here. The error type itself is a standalone/concrete type. If users want, they can introduce a new error type based on it.

Reply:
+1 to say based on the error type.